.github/workflows/smoke-test.yaml

```diff
@@ -172,6 +172,6 @@ jobs:
       - name: Dump connectivity related logs and events
         if: ${{ failure() }}
         run: |
-          kubectl get pods -o wide
-          kubectl get deploy -o wide
+          kubectl describe pods
```

Comment:
can you help to update for https://github.com/cilium/cilium/blob/master/.github/workflows/smoke-test.yaml#L84 and https://github.com/cilium/cilium/blob/master/.github/workflows/smoke-test-ipv6.yaml#L86 for consistency and just in case ? Thanks a lot :100:

Reply:
Done. Thanks for the review!